### PR TITLE
Add triad

### PR DESCRIPTION
**Pull request by rdmolony**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

---

@kvnkho I made a stab at getting triad ready for conda-forge, are you happy to be listed as a maintainer?

### CODE REVIEW

**Comment by rdmolony** [comment]
only took 4 goes to get the python version specified :)

**Comment by rdmolony** [comment]
Ref fugue-project/triad#70

File: recipes/triad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  run:
+    - python >=3.6
+    - pandas
+    - six
+    - ciso8601
+    - pyarrow
+    - fs
+
+test:
```
Comment (dopplershift) [inline recipes/triad/meta.yaml:34]: Would be good to add an import test:
```suggestion
test:
  imports:
    - triad
```

File: recipes/triad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (dopplershift) [inline recipes/triad/meta.yaml:5]: Go ahead and remove these comments from the template:
```suggestion
```

File: recipes/triad/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/fugue-project/triad
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'A collection of python utility functions for Fugue projects'
+  # The remaining entries in this section are optional, but recommended.
```
Comment (dopplershift) [inline recipes/triad/meta.yaml:45]: And remove this too:
```suggestion
```

File: recipes/triad/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  doc_url: https://triad.readthedocs.org
+  dev_url: https://github.com/fugue-project/triad
+
+extra:
+  recipe-maintainers:
+    - kvnkho
```
Comment (dopplershift) [inline recipes/triad/meta.yaml:51]: Why aren't you listing yourself here?

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Comment by rdmolony** [comment]
@goodwanghan I added you in as a maintainer, I hope this is okay with you?

**Comment by rdmolony** [comment]
thanks for your help @dopplershift

**Comment by goodwanghan** [comment]
This is great! Thank you!

**Comment by rdmolony** [comment]
@conda-forge-admin ready for review! Thanks for all your help @dopplershift!

**Comment by rdmolony** [comment]
@conda-forge-admin ready for review!

**Comment by rdmolony** [comment]
@conda-forge/help-python would you mind having a look at this PR?  I have made the suggested changes but I'm not sure if they have registered

**Pull request merged**